### PR TITLE
Move ad hoc-related translations to [Networking]

### PR DESCRIPTION
**Pull request by vnctdj**

### CODE REVIEW

**Comment by vnctdj** [comment]
I will update en_US.ini, as always :)

**Comment by vnctdj** [comment]
Still no reply... Is there a problem with my pull request ? :-/
@hrydgard

**Comment by hrydgard** [comment]
sorry been a bit busy. Seems fine :)

File: UI/GameSettingsScreen.cpp [hunk 1]
```diff
@@ -382,7 +382,7 @@ void GameSettingsScreen::CreateViews() {
 	LinearLayout *networkingSettings = new LinearLayout(ORIENT_VERTICAL);
 	networkingSettings->SetSpacing(0);
 	networkingSettingsScroll->Add(networkingSettings);
-	tabHolder->AddTab(s->T("Networking"), networkingSettingsScroll);  // TODO: This string was originally just an item header so in the wrong category
+	tabHolder->AddTab(n->T("Networking"), networkingSettingsScroll);
```
Comment (hrydgard) [inline UI/GameSettingsScreen.cpp]: For consistency we should take the title out of "ms" here.

Comment (vnctdj) [inline UI/GameSettingsScreen.cpp]: I'm not sure to understand, do you mean we should use "ms" instead of "n" for the title ?

**Comment by vnctdj** [comment]
Is my new commit correct, @hrydgard ? :)

**Comment by hrydgard** [comment]
Yeah, seems fine. Now we just need to update all the other languages...

**Comment by vnctdj** [comment]
Thank you for merging :)
I will see what I can do for other languages :-/

**Pull request merged**